drivers/network/dd/e1000/nic.h

```diff
@@ -86,6 +86,11 @@ typedef struct _E1000_ADAPTER
     NDIS_PHYSICAL_ADDRESS ReceiveBufferPa;
     ULONG ReceiveBufferEntrySize;
 
+
+    /* EEPROM/nvm stuff */
+    ULONG E1000_EERD_DONE_BIT;
```

Comment:
I'd recommend naming these something that doesn't look like a macro.

Reply:
Done